src/modules/admin/admin-categories/AdminCategories.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback } from 'react';
+import { useRouteMatch, useHistory } from 'react-router-dom';
+
+import { Button as MuiButton } from '@material-ui/core';
+import UnfoldMoreIcon from '@material-ui/icons/UnfoldMore';
+
+import { Button } from 'ui';
+
+import { Url } from 'utils';
+
+import { Category } from '../models';
```

Comment:
models and utils should be just files in root module dir. Change import path to .. after add models and utils to root index.ts